applications/googlehome-meets-dotnetcontainers/GoogleHomeAspNetCoreDemoServer/appsettings.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "GoogleCloudSettings": {
+    "ProjectId": "<YOUR PROJECT ID>",
```

Comment:
By convention, we use hyphenated place holders: YOUR-PROJECT-ID

Reply:
Fixed